### DISCUSSION TITLE
Unable to setup the fade-in

### DISCUSSION THREAD

**Discussion by Rocco83** [senshu/Sozi#657] [Q&A]
Hi,

I'm having trouble in setting up the fade-in transition.
Looking at
https://github.com/sozi-projects/Sozi/issues/175
I was doing the following:
0) add the "HiddenText" layer in sozi
1) have a frame with the layer "HiddenText" to full transparency
2) copy the frame, rename etc
3) select the HiddenText layer and change it to full opacity

But, seems that i'm unlucky.

FWIW, this was working before.
After moving the frame, i was having the error.
But i've also removed the frame and added it again.

Thank you very much,
Daniel

**Comment by aumouvantsillage** [comment]
I'd be glad to help, but I don't have enough information about your problem.

> But, seems that i'm unlucky.
> After moving the frame, i was having the error.

I understand that you want to create a "fade-in" transition and it seems that you followed the appropriate steps, but you don't explain what actually happens when you try to play the presentation.
When you say "I was having the error", what are you referring to?

**Reply by Rocco83** [reply]
I'm sorry if i was not clear enough.
"the error" means that the transition is not "fade in", but rather "something else".

Probably the best is to see it:
transition from:
https://www.champ1foundation.eu/2022-tommy-presentation.sozi.html#wishes
to:
https://www.champ1foundation.eu/2022-tommy-presentation.sozi.html#wishes2

Hope that this clarify the question.
